src/controllers/index.controller.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response } from "express";
+
+import * as config from "../config";
+
+export const startController = (req: Request, res: Response) => {
```

Comment:
we have typically used 'get' and 'post' for the function names in the controllers - eg - https://github.com/companieshouse/confirmation-statement-web/blob/main/src/controllers/start.controller.ts
how do you feel about using that approach?  I think it makes it easier to read as we can easily see what function is called for the get and what is called for the post.

Reply:
here's the original bug - https://companieshouse.atlassian.net/browse/NCS-290

Reply:
It sounds good to me, will do